src/odemis/acq/fastem_conf.py

```diff
@@ -91,6 +91,15 @@ def configure_scanner(scanner, mode):
     # Immersion needs to be set before changing the horizontalFoV, as the range is updated
     scanner.immersion.value = conf["immersion"]
 
+    if scanner.immersion.value is False:
+        # Shift the overview image by a calibrated value, to align it with the live view.
```

Comment:
The comment is very specific for overview and live mode. However, here you just apply a shift in case the immersion mode is switched off (=field free mode). I suggest to rephrase in order to be more generic:
'Correct for shift in image between immersion mode and field free mode.'